examples/Cassie/run_simple_sim_with_cassie_sensor.cc

```diff
@@ -0,0 +1,237 @@
+#include <memory>
```

Comment:
Question--can this replace the basic simulation, instead of being added as a separate file? Would there be a downside to that?

Reply:
I replaced the basic simulation with my simulation with sensor. 
I prefer combining them into one file, so that we could reduce repeated code. I guess the downside is just that run_simple_sim.cc is now slight longer than before, and it puts more burden on the people who read it for the first time (but I think it's fine).